### ISSUE TITLE
[A11Y] Text field keyboard navigation

### ISSUE THREAD

**Issue by erikkroes** [Tiqr/eduid-app-android#261]
## Finding
Some text fields kan be entered with the tab-key. However, they can not be exited with the tab-key. Only with shift+tab.

## Location

Several text fields like these

<img width="661" height="527" alt="" src="https://github.com/user-attachments/assets/cdc98e37-44a4-491f-9244-e3881fe66111" />

## Problem

People should be able to operate the app with common keyboard shortcuts. When they can't, the app becomes unusable for all people using keyboards, and devices like them (which are many).

## Solution

Implementation is unclear, but it might need to be updated to a native text field.

## Resources
- https://www.w3.org/WAI/WCAG22/Understanding/keyboard.html
- https://www.w3.org/WAI/WCAG22/Understanding/no-keyboard-trap.html
